qa/tasks/cephfs/test_volumes.py

```diff
@@ -162,6 +162,15 @@ def test_subvolume_create_with_auto_cleanup_on_fail(self):
         else:
             raise
 
+    def test_subvolume_create_with_invalid_size(self):
+        # create subvolume with an invalid size -1
+        subvolume = self._generate_random_subvolume_name()
+        try:
+            self._fs_cmd("subvolume", "create", self.volname, subvolume, "--size", -1)
+        except CommandFailedError as ce:
+            if ce.exitstatus != errno.ENOENT:
+                raise
```

Comment:
Also add,
```
else:
        raise
```

The test should error out if no exception is raised

Reply:
Yes, we can raise an exception here. But using raise without arguments shows an error "TypeError: exceptions must be old-style classes or derived from BaseException, not NoneType". Please see https://stackoverflow.com/a/27947646

Reply:
Good point. Raise RuntimeError with an error message that you expected fs subvolume create to fail here?
I think there are other places in this code where we incorrectly use `raise` when not re-raising Exceptions. We should clean that up in this or a later PR. Maybe just file a tracker ticket for now?
```
else:
    raise RuntimeError("expected the `fs subvolume create` command to fail")
 ```

Reply:
I already have a PR that was closed. I will re-open it and will do the needful.